client/app/views/app_layout.coffee

```diff
@@ -14,9 +14,9 @@ require 'normalize.css/normalize.css'
 
 module.exports = class AppLayout extends LayoutView
 
-    template: require './templates/layout_app'
+    template: -> '<main/>'
```

Comment:
Backbone's views `template` property expect a function that can be call at rendering, and which returns a string containing the HTML to inject. Here, I've prefer to create a simple function that return my unique tag instead of a separated template file.

Also, I create a card in our technical debt, coz the AppLayout isn't realy relevant anymore and we can drop it soon 😉.